File: tools-excel/src/main/java/cn/gjing/tools/excel/MetaStyle.java
package cn.gjing.tools.excel;

import lombok.*;
import org.apache.poi.ss.usermodel.CellStyle;

/**
 * Excel meta style
 *
 * @author Gjing
 **/
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MetaStyle {
    private CellStyle headStyle;
    private CellStyle bodyStyle;
    private CellStyle titleStyle;
}

File: tools-excel/src/main/java/cn/gjing/tools/excel/MetaStyle.scala
package cn.gjing.tools.excel

import org.apache.poi.ss.usermodel.CellStyle

import scala.beans.BeanProperty

/**
 * Style metadata
 * @author Gjing
 **/
class MetaStyle {
  @BeanProperty
  var headStyle:CellStyle = _
  @BeanProperty
  var bodyStyle:CellStyle = _
  @BeanProperty
  var titleStyle:CellStyle = _

  def this(headStyle:CellStyle,bodyStyle:CellStyle,titleStyle:CellStyle){
    this()
    this.headStyle = headStyle
    this.bodyStyle = bodyStyle
    this.titleStyle = titleStyle
  }
}
